client/build_project.sh

```diff
@@ -21,19 +21,19 @@ function _oc() {
     oc --config=./node.kubeconfig $@
 }
 
-NAME=$1
-TAG=$2
+APPID=$1
+JOBID=$2
 REPO=$3
 REPO_BRANCH=$4
 REPO_BUILD_PATH=$5
 TARGET_FILE=$6
 NOTIFY_EMAIL=$7
-DEPENDS_ON=$8
+DESIRED_TAG=$8
+DEPENDS_ON=$9
```

Comment:
depends on can have multiple values in array and this was causing issue while parsing desired tag